resources/test.json

```diff
@@ -54,6 +54,23 @@
             "test": "paratest --version",
             "tags": ["test"]
         },
+        {
+            "name": "pest",
+            "summary": "The elegant PHP Testing Framework",
+            "website": "https://github.com/pestphp/pest",
+            "sh": {
```

Comment:
Something didn't go right here. Both `sh` and `composer-bin-plugin` should be nested inside `command`. Take a look at the earlier example I referred to: https://github.com/jakzal/toolbox/blob/1ef9e2f185afb5a208f3af214a5b77459e68c2a6/resources/phpcs.json#L21C1-L29